app/aws/s3.py

```diff
@@ -107,3 +107,19 @@ def upload_letters_pdf(reference, crown, filedata):
 
     current_app.logger.info("Uploading letters PDF {} to {}".format(
         upload_file_name, current_app.config['LETTERS_PDF_BUCKET_NAME']))
+
+
+def get_list_of_files_by_suffix(bucket_name, subfolder='', suffix=''):
+    s3_client = client('s3', current_app.config['AWS_REGION'])
+    paginator = s3_client.get_paginator('list_objects_v2')
+
+    page_iterator = paginator.paginate(
+        Bucket=bucket_name,
+        Prefix=subfolder
+    )
```

Comment:
We can add a parameter here to only get the files added since "today"

Reply:
There is no filter for returned list by date. The field StartAfter takes in the file name and returns the files after that particular file. 
I have added filtering in the function itself to process the files and only return files that are modified after the date, using the meta data of the files returned from paginate.